### PR TITLE
Custom RPK

### PR DESCRIPTION
**Pull request by tanaaKa**
When merged will add:

RPK that only takes custom 45-round RPK magazine(s).

### CODE REVIEW

**Comment by tanaaKa** [comment]
Good to send it

File: addons/fnf_rpk/CfgMagazines.hpp [hunk 1]
```diff
@@ -0,0 +1,12 @@
+class CfgMagazines {
+    class rhs_45Rnd_545X39_AK_Green;
+	class fnf_45rnd_rpk_green: rhs_45Rnd_545X39_AK_Green {
+		scope = 1;
+		author = "tanaKa";
+		count = 45;
+		descriptionShort = "Caliber: 5.45x39mm<br/>Rounds: 45<br/>Used in: RPK";
+		displayName = "57-N-231P 45 round mag";
```
Comment (Mallen-git) [inline addons/fnf_rpk/CfgMagazines.hpp:8]: that is definitly not 5.45x39mm ammo name right there XD, should be something like 7N22 or 7N10

Comment (tanaaKa) [reply addons/fnf_rpk/CfgMagazines.hpp:8]: Will update tomorrow if I get time

**Comment by tanaaKa** [comment]
Ready to go @indig0fox

**Pull request merged**